internal/agentdeployer/agent.go

```diff
@@ -74,9 +75,10 @@ var _ AgentDeployer = new(DockerComposeAgentDeployer)
 type dockerComposeDeployedAgent struct {
 	agentInfo AgentInfo
 
-	ymlPaths []string
-	project  string
-	env      []string
+	ymlPaths  []string
+	project   string
+	env       []string
+	configDir string
```

Comment:
Moved from agentInfo struct to be just defined here in the deployed agent.

Currently, this variable is not used outside this struct `dockerComposeDeployedAgent`, so it is not needed to keep it afterwards